### PR TITLE
Add profiles for other linux platforms

### PR DESCRIPTION
**Pull request by yselkowitz**

### CODE REVIEW

File: linux-aarch64/Makefile [hunk 1]
```diff
@@ -0,0 +1,17 @@
+CC = cc
```
Comment (fl4via) [inline linux-aarch64/Makefile]: Please, add the license header to the Makefiles as well

**Comment by fl4via** [comment]
@yselkowitz thanks for your PR! Can you please add a license header to the Makefiles?

**Comment by yselkowitz** [comment]
@fl4via since none of the existing Makefiles have such, why is this necessary for these?  And how exactly do you want them to look?

**Comment by fl4via** [comment]
@yselkowitz you're absolutely right, please disregard. Thanks for your PR!

**Pull request merged**